README.md

```diff
@@ -32,11 +32,37 @@ goose session start
 You will see a prompt `G❯`:
 
 ```
-G❯ type your instructions here exactly as you would tell a developer.
+G❯ Looking at the in progress changes in this repo, help me finish off the feature. CONTRIBUTING.md shows how to run the tests.
+```
+
+> [!NOTE]
+> From here you can talk directly with goose - send along your instructions. If you are looking to exit, use `CTRL+D`,
+> although goose should help you figure that out if you forget. More examples:
```

Comment:
nit: like above, i'd move all the examples inside the note i think (or alternatively all outside)

Reply:
yep, have moved it to an example section, makes sense